### PR TITLE
Improve tabstrip appearance

### PR DESCRIPTION
**Pull request by rock3r**
* Remove a ton of unused resources
* Fix the tabs' text and indicator styling
   * Setting the text typeface implies a hack I am not happy with but it seems the only way to avoid most other TextViews' contents from going totally blank 😨 
* Min SDK 21 and that's it
* IDE Lint settings tweaks

TODO:
* fix the shadow on top of the appbar layout (elevation and insets woes)

 Before | After
 --- | ---
 image | image

### CODE REVIEW

File: app/src/main/java/net/squanchy/schedule/view/ScheduleView.java [hunk 1]
```diff
@@ -32,11 +36,31 @@ protected void onFinishInflate() {
...
+        // results in a bunch of other views on screen to stop drawing their text, for reasons only
+        // known to the Gods of Kobol. We can't theme things in the TabLayout either, as the
+        // TextAppearance is applied _after_ inflating the tab views, which means Calligraphy can't
+        // intercept that either. Sad panda.
+        tabLayout.addOnLayoutChangeListener((v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom) -> {
+            Typeface typeface = TypefaceUtils.load(getContext().getAssets(), "fonts/LeagueSpartan-Bold.otf");
+            int tabCount = tabLayout.getTabCount();
+            for (int i = 0; i < tabCount; i++) {
+                TabLayout.Tab tab = tabLayout.getTabAt(i);
+                if (tab == null) {
```
Comment (danybony) [inline app/src/main/java/net/squanchy/schedule/view/ScheduleView.java:56]: When can this happen?

**Review by danybony — COMMENTED** [review]
Nice hack and yayyyy for minsdk 21.
Just one question and then good to go

File: app/src/main/java/net/squanchy/schedule/view/ScheduleView.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/net/squanchy/schedule/view/ScheduleView.java:56]: Not sure but it's annotated as `@Nullable` and there's a ternary that can return null inside, so I went for it

Comment (danybony) [reply app/src/main/java/net/squanchy/schedule/view/ScheduleView.java:56]: Weird, but better safe than sorry 👍

**Review by danybony — APPROVED** [review]

**Pull request merged**